src/antlr/antlr.test.ts

```diff
@@ -0,0 +1,137 @@
+import { CharStreams, CommonTokenStream } from "antlr4ts";
+import { UppercaseCharStream } from "./UppercaseCharStream";
+import { SqlBaseLexer } from "./SqlBaseLexer";
+import { SqlBaseParser } from "./SqlBaseParser";
+import { getLineage } from "./LineageUtil";
+import { Lineage, Table } from "../Lineage";
+
+type ColumnId = string;
+const columnsMapping: { [tableId: string]: ColumnId[] } = {
+  account: ["account_type", "account_id", "account_description", "account_parent", "account_rollup"],
+  employee: [
+    "birth_date",
+    "department_id",
+    "education_level",
+    "employee_id",
+    "end_date",
+    "first_name",
+    "full_name",
+    "gender",
+    "hire_date",
+    "last_name",
+    "management_role",
+    "marital_status",
+    "position_id",
+    "position_title",
+    "salary",
+    "store_id",
+    "supervisor_id"
+  ],
+  salary: [
+    "currency_id",
+    "department_id",
+    "employee_id",
+    "overtime_paid",
+    "pay_date",
+    "salary_paid",
+    "vacation_accrued",
+    "vacation_used"
+  ]
+};
+
+const getTable = (id: string) => ({
+  id
+});
+
+const getColumns = (tableId: string) => {
+  const columns = columnsMapping[tableId] || [];
+  return columns.map(columnId => ({
+    id: columnId,
+    tableId
+  }));
+};
+
+const metadataProvider = {
+  getTable: getTable,
+  getColumns: getColumns
+};
+
+// function to log lineage when testing
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function logLineage<TableData extends { id: string }, ColumnData extends { id: string }>(
+  lineage: Lineage<TableData, ColumnData>
+) {
+  console.log(JSON.stringify(lineage, undefined, 2));
+}
```

Comment:
Let's remove this and add real unit tests

Reply:
Removed (it was not used). What do you mean by real tests? We have tests, I can add full result check but it's not very robust because some parts like identifiers and the order of elements can change.